Neos.Fusion/Classes/FusionObjects/Internal/PrivateComponentPropsImplementation.php

```diff
@@ -0,0 +1,33 @@
+<?php
+namespace Neos\Fusion\FusionObjects\Internal;
```

Comment:
new namespace for internal things.

i choose this, so when using IDEs to write fusion, one wont accidentally choose `Neos.Fusion:Internal.PrivateComponentProps`

so im against:

- `Neos.Fusion:PrivateComponentProps`
- `Neos.Fusion:Helpers.PrivateComponentProps`